src/libs/DateUtils.js

```diff
@@ -15,13 +14,15 @@ Onyx.connect({
  * Gets the user's stored time-zone NVP and returns a localized
  * Moment object for the given timestamp
  *
+ * @param {String} locale
  * @param {Number} timestamp
  *
  * @returns  {Moment}
  *
  * @private
  */
-function getLocalMomentFromTimestamp(timestamp) {
+function getLocalMomentFromTimestamp(locale, timestamp) {
+    moment.locale(locale);
```

Comment:
If I understand correctly, the `modal.locale` method takes a language (like English = "en"). Calling the variable that we pass to it `locale` is confusing.

I think we should rename the variable name `locale` to `lang` just for clarity. That way, when you're calling something like `timestampToDateTime` you know the first argument that you have to pass is a language and not a location.

Reply:
No, locale is the better name, since this is not a language, but a locale, which is something like `en-US`.

Reply:
Really? Because we are only setting it as `'en'`.
```
{DateUtils.timestampToDateTime('en', props.timestamp)}
```

Either way, it's not a biggie. I just think it will end up being a bit confusing.

Reply:
Yeah, that's just an example so this PR could be tested. We will be replacing that call with the preferred locale later.